build.gradle

```diff
@@ -20,6 +20,16 @@
 buildscript {
     ext {
         jacocoVersion = '0.8.5'
+        retrofitVersion = '2.9.0'
+        okhttpVersion = '4.7.2'
+        oltuVersion = '1.0.1'
```

Comment:
This seems curious, to me.. isn't this RUNTIME related, not BUILD time? Why is e.g. okhttp required to generate code? There's probably a good reason. But we didn't seem to have it before, so I thought I could ask.

Reply:
OKHttp doesn't generate anything, but we are still building the client JAR... so unless I'm missing something here seems to be normal to me...

Before it was not defined in Fineract's dependency management section, because the build was driven by a totally independent - single project - Maven pom.xml that had no ties to the fineract project setup (in terms of dependencies).

These dependencies are just a consequence of having fineract-client as a module.